### ISSUE TITLE
How to test this repository?

### ISSUE THREAD

**Issue by Hywan** [hoaproject/Infrastructure#2]
Fire!

I guess we are going to use Travis on this one. Any idea how?

**Comment by jubianchi** [comment]
good luck with building images on travis: builds have timeouts and you are likely to reach them while waiting for the box to get up and open SSH...

**Comment by Hywan** [comment]
@jubianchi: We can set the SSH timeout to `1000000s` when building with Travis (`packer build -var … Template.json`). I don't think timeouts would be an issue… or maybe you did try?

**Comment by jubianchi** [comment]
@Hywan yes I tried this a while ago and reached timeouts. If the build does not produce output or the main process is idle for too long, the build gets cancelled by travis.

**Comment by Hywan** [comment]
@jubianchi Hmm damned…

**Comment by Hywan** [comment]
To test Shell scripts: https://github.com/sstephenson/bats. Thanks @jubianchi for the link!

**Comment by Hywan** [comment]
@jubianchi Do you think it would be possible on CircleCI?
